install_with_docker.sh

```diff
@@ -150,9 +150,9 @@ start_docker
 
 echo -e "\n===> Pulling the latest docker image for SigLens"
 
-wget "https://github.com/siglens/siglens/releases/download/${SIGLENS_VERSION}/server.yaml"
+curl -O -L "https://github.com/siglens/siglens/releases/download/${SIGLENS_VERSION}/server.yaml"
 $sudo_cmd docker pull siglens/siglens:${SIGLENS_VERSION}
-$sudo_cmd mkdir data
+mkdir data
```

Comment:
If data already exists , this will log a failure. Pass the -p option or whatever that option is which does not complain if dir already exists